src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -539,6 +541,60 @@ export class QuickOpenController extends Component implements IQuickOpenService
 		}, 100 /* to prevent flashing, we accumulate visibility changes over a timeout of 100ms */);
 	}
 
+	private getSelectedText(): string | null {
+		const activeEditor = this.editorService.getActiveEditor();
+
+		if (!activeEditor) {
+			return null;
+		}
+
+		const editorControl = activeEditor.getControl() as IEditor | null;
+
+		if (!editorControl) {
+			return null;
+		}
+
+		const selection = editorControl.getSelection();
+
+		if (!selection) {
+			return null;
+		}
+
+		const startPosition = selection.getStartPosition();
+		const endPosition = selection.getEndPosition();
+
+		let isSelection = startPosition.column !== endPosition.column;
+		let isMultiline = startPosition.lineNumber !== endPosition.lineNumber;
+
+		if (!isSelection || isMultiline) {
+			return null;
+		}
+
+		const model = editorControl.getModel();
+
+		if (!('getValueInRange' in model)) {
+			// not interested in any other type of model, but TextModel
+			return null;
+		}
+
+		let selectedText = (model as ITextModel).getValueInRange(selection);
+
+		return selectedText;
+	}
+
+	private shouldPrefill(prefix?: string): boolean {
+		const isTurnedOn = this.configurationService.getValue('workbench.quickOpen.prefillFromSelection') as string;
+		const prefixes = this.configurationService.getValue('workbench.quickOpen.prefillPrefixes') as Array<string | null>;
+
+		if (!isTurnedOn) {
+			return false;
+		}
+
+		const prefillsPrefix = prefixes.indexOf(prefix) !== -1;
+
+		return prefillsPrefix;
```

Comment:
Could've been one line? 
`return prefixes.indexOf(prefix) !== -1;`

Reply:
yeah

corrected to be one line